src/amo/components/AddonAdminLinks/index.js

```diff
@@ -0,0 +1,157 @@
+/* @flow */
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import {
+  ADDONS_CONTENTREVIEW,
+  ADDONS_EDIT,
+  ADDONS_POSTREVIEW,
+  ADMIN_TOOLS_VIEW,
+  THEMES_REVIEW,
+} from 'core/constants';
+import translate from 'core/i18n/translate';
+import { hasPermission } from 'amo/reducers/users';
+import type { AddonType } from 'core/types/addons';
+import { isTheme } from 'core/utils';
+import DefinitionList, { Definition } from 'ui/components/DefinitionList';
+import type { I18nType } from 'core/types/i18n';
+import type { AppState } from 'amo/store';
+
+type Props = {|
+  addon: AddonType | null,
+|};
+
+type InternalProps = {|
+  ...Props,
+  i18n: I18nType,
+  hasAdminPermission: boolean,
+  hasCodeReviewPermission: boolean,
+  hasContentReviewPermission: boolean,
+  hasEditPermission: boolean,
+  hasThemeReviewPermission: boolean,
+|};
+
+export class AddonAdminLinksBase extends React.Component<InternalProps> {
+  render() {
+    const {
+      addon,
+      hasAdminPermission,
+      hasCodeReviewPermission,
+      hasContentReviewPermission,
+      hasEditPermission,
+      hasThemeReviewPermission,
+      i18n,
+    } = this.props;
+
+    if (addon === null) {
+      return null;
+    }
+
+    const showCodeReviewLink = hasCodeReviewPermission && !isTheme(addon.type);
+    const showThemeReviewLink = hasThemeReviewPermission && isTheme(addon.type);
+    const hasALink =
+      hasEditPermission ||
+      hasAdminPermission ||
+      hasContentReviewPermission ||
+      showCodeReviewLink ||
+      showThemeReviewLink;
+
+    if (!hasALink) {
+      return null;
+    }
+
+    const editLink = hasEditPermission ? (
+      <li>
+        <a
+          className="AddonAdminLinks-edit-link"
+          href={`/developers/addon/${addon.slug}/edit`}
+        >
+          {i18n.gettext('Edit add-on')}
+        </a>
+      </li>
+    ) : null;
+
+    const adminStatusLink = hasAdminPermission ? (
+      <li>
+        <a
+          className="AddonAdminLinks-admin-status-link"
+          href={`/admin/addon/manage/${addon.slug}/`}
+        >
+          {i18n.gettext('Admin add-on status')}
+        </a>
+      </li>
+    ) : null;
+
+    const adminLink =
+      hasAdminPermission && hasEditPermission ? (
+        <li>
+          <a
+            className="AddonAdminLinks-admin-link"
+            href={`/admin/models/addons/addon/${addon.id}`}
+          >
+            {i18n.gettext('Admin add-on')}
+          </a>
+        </li>
+      ) : null;
+
+    const contentReviewLink = hasContentReviewPermission ? (
+      <li>
+        <a
+          className="AddonAdminLinks-contentReview-link"
+          href={`/reviewers/review-content/${addon.slug}`}
+        >
+          {i18n.gettext('Content review add-on')}
+        </a>
+      </li>
+    ) : null;
+
+    const codeReviewLinkText = isTheme(addon.type)
+      ? i18n.gettext('Review theme')
+      : i18n.gettext('Code review add-on');
```

Comment:
*Review add-on code*, maybe?

Reply:
That does sound better to me, but the text comes directly from https://github.com/mozilla/addons-frontend/issues/4632#issue-308115330. I know that "code review" is a thing, so maybe the current value of "Code review add-on" is still preferred.

@jvillalobos Which do you think is a better label for the link:
* Code review add-on
* Review add-on code

Reply:
I like "Review add-on code"